packages/eas-cli/src/credentials/ios/utils/printCredentials.ts

```diff
@@ -66,28 +71,46 @@ export function displayIosCredentials(
   const projectFullName = `@${app.account.name}/${app.projectName}`;
   const isMultitarget = targets.length > 1;
 
-  Log.log(chalk.bold(`iOS Credentials`));
-  Log.addNewLineIfNone();
-  Log.log(`  Project: ${chalk.bold(projectFullName)}`);
+  const fields = [
+    { label: 'iOS Credentials', value: '' },
+    { label: 'Project', value: projectFullName },
+  ];
+  Log.log(formatFields(fields, { labelFormat: chalk.cyan.bold }));
 
   for (const { targetName, bundleIdentifier } of targets) {
     if (isMultitarget) {
       Log.newLine();
-      Log.log(`  Target: ${chalk.bold(targetName)}`);
+      Log.log(
+        formatFields([{ label: 'Target', value: targetName }], { labelFormat: chalk.cyan.bold })
+      );
     }
-    Log.log(`  Bundle Identifier: ${chalk.bold(bundleIdentifier)}`);
+    Log.log(
+      formatFields([{ label: 'Bundle Identifier', value: bundleIdentifier }], {
+        labelFormat: chalk.cyan.bold,
+      })
+    );
```

Comment:
`formatFields()` should be called once for each "field group", because it aligns values to be in the same column, basing on the longest label.

So if should be sth like:
```ts
const fields = [];
if (isMultitarget) {
  fields.push({ label: 'Target', value: targetName });
}
fields.push({ label: 'Bundle Identifier', value: bundleIdentifier });
// ... rest of fields from this group
formatFields(fields, { labelFormat: chalk.cyan.bold });
```
Probably that's why the first three fields (project, Bundle identifier, Apple team) are misaligned on the screenshot. 😉

Reply:
![Screen Shot 2021-09-29 at 2 01 41 PM](https://user-images.githubusercontent.com/6380927/135348411-0865b048-4d94-415b-b5f5-a2b4bf20b005.png)

I've updated the PR to push everything to the `fields` array, and do `Log.log(formatFields(fields, { labelFormat: chalk.cyan.bold }));` at the very end of the function. One challenge is handling the logs of new lines -- we originally did this with `Log.newLine()`. But now, we have to track this in `fields`, otherwise all the new lines will get printed first. Kind of hacky, but I've just replaced them all with `fields.push({label:'', value: ''})`

Reply:
It looks like the best solution is something in the middle:
```ts
// pseudo-code
Log.log('iOS credentials');
iosFields.push(project);
iosFields.push(bundleId);
iosFields.push(appleTeam);
Log.log(formatFields(iosFields);

Log.newLine();
Log.log('Push key');
pushFields.push(devPortalId);
// ...
Log.log(formatFields(pushFields);

Log.newLine();
// the same for each "field group"
```

If any field or title is too long, I liked your hack from Android:
```ts
fields.push({ label: 'some very long long label', value: '' });
fields.push({ label: '', value: 'a longish value here' });
```

Reply:
Ah, I'm sorry, now I see what you mean 😅 I shouldn't have review PRs right after I wake up.

so the problem was with multitarget project? then we have 2 options:
- try with separate group for each target. Target name can (but does not have to) be then displayed as normal `Log.log()`
- leave your newline hack, I don't mind leaving it as is